recipes/mpir/bld.bat

```diff
@@ -0,0 +1,19 @@
+cd build.vc14
```

Comment:
Let's try the hacky Python version check for now and then this can always be improved on in the feedstock. Personally am eager to see this one go out. :smile: